src/Annotation/ApiResource.php

```diff
@@ -274,128 +312,127 @@ final class ApiResource
      *
      * @var array
      */
+    private $openapiContext;
+
+    /**
+     * @see https://github.com/Haehnchen/idea-php-annotation-plugin/issues/112
+     * @see https://api-platform.com/docs/core/default-order/#overriding-default-order
+     *
+     * @var array
+     */
     private $order;
 
     /**
      * @see https://github.com/Haehnchen/idea-php-annotation-plugin/issues/112
      *
+     * @var string|false
+     */
+    private $output;
+
+
+    /**
+     * @see https://github.com/Haehnchen/idea-php-annotation-plugin/issues/112
+     * @see https://api-platform.com/docs/core/pagination/#globally-1
```

Comment:
Wrong link, it should be https://api-platform.com/docs/core/pagination/#for-a-specific-resource-1